ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -8,3 +8,34 @@
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
 def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None):
     return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
+
+
+# batch_norm
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def batch_norm(
+    x,
+    running_mean,
+    running_var,
+    weight,
+    bias,
+    training=False,
+    momentum=0.9,
+    epsilon=1e-05,
+    data_format="NCHW",
+    use_global_stats=None,
+    name=None,
+):
+    return ivy.batch_norm(
+        x,
+        running_mean,
+        running_var,
+        weight,
+        bias,
+        training,
+        momentum,
+        epsilon,
+        data_format,
```

Comment:
You need to add some logic to map the `data_format` parameter here with the format that `ivy.batch_norm` expects. Basically, the `data_format` parameter in ivy can be either `NCS` or `NSC` ( channel second or channel last respectively) while in paddle, it can have the following values:
```
“NC”, “NCL”, “NCHW”, “NCDHW”, “NLC”, “NHWC” or “NDHWC”, 
```
You can easily observe that that first 3 map to `NCS` while the last 2 map to `NSC`

Reply:
Yup, it's the former 
```python
if data_format in [“NC”, “NCL”, “NCHW”, “NCDHW”]: # Map to 4 first values
    return ivy.batch_norm with data_format parameter as "NCS"
elif data_format_paddle in [“NLC”, “NHWC", “NDHWC”]: # Map to last 3 values
return ivy.batch_norm with data_format parameter as "NSC"
else:
raise ValueError('Invalid data_format value')